CodeStarterCamp_Week3/step2.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  step2.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by 홍세희 on 2023/09/12.
+//
+
+enum Coffee: String {
+    case americano = "아메리카노"
+    case cafeLatte = "카페라떼"
+    case cappuccino = "카푸치노"
+    case mocha = "모카"
+    case prapucihno = "프라푸치노"
+    case banillaLatte = "바닐라라떼"
+}
+
+struct Person {
+    var money: Int
+    var name: String
+
+    func order(_ coffee: Coffee, of coffeeShop: CoffeeShop, by name: String) {
+        if let price = menu[coffee] {
+            if money < price {
+                print("잔액이 \(price - money)원 만큼 부족합니다. ")
+            }
+            else {
+                coffeeShop.make(_: coffee, from: name)
+                coffeeShop.sales += price
+            }
+        }
+    }
+}
+
+class CoffeeShop {
+    var sales: Int
+    var barista: Person
+    var menu: [Coffee: Int]
+    var pickUpTable: [String]
+    
+    init(sales: Int, barista: Person, menu: [Coffee: Int], pickUpTable: [String]) {
```

Comment:
pickUpTable의 경우 초기값을 빈 배열로 할당해두셔도 좋을 것 같습니다. 🤔
(주문이 없는데도 커피가 올려져있을 경우가 없다면말이죠 💭)
```suggestion
    var pickUpTable: [Coffee] = []
    
    init(sales: Int, barista: Person, menu: [Coffee: Int]) {
```